libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -1,5 +1,16 @@
 <ng-container *ngIf="(readingList$ | async).length > 0; else empty">
-  <div class="reading-list-item" *ngFor="let b of readingList$ | async">
+  <div class="reading-list-item" data-testing="reading-listitem" *ngFor="let b of readingList$ | async">
+      <div>
+          <button
+            mat-icon-button
+            [matTooltip]="b.finished ? 'Mark as Unfinished' : 'Mark as Finished'"
+            data-testing="mark-book-as-finished"
+            [color]="b.finished ? 'alert' : null"
+            [attr.aria-label]="'Mark ' + b.title + ' as finished'"
```

Comment:
I like your UX solution here! It's creative and intuitive. The only thing that needs to be fixed up here is that the aria label should use the same state logic as the tooltip